README.md

```diff
@@ -7,16 +7,18 @@
 ```python
 print('what to print')
 ```
+- output :<br> 
+what to print
```

Comment:
hasil outputny dimasukin k code block be, carony lihat di [sini](https://github.com/airlab-unsri/basic-python-course/pull/6#issuecomment-551373493). Terus `<br>`-ny dihapus be setelah itu.